Extensions/retags.js

```diff
@@ -181,7 +217,12 @@ XKit.extensions.retags = {
 				});
 			}
 			$t.addClass(cls);
-			$c.append($retags);
+			if (!isPopover) {
+				$c.append($retags);
+				return;
+			}
+			$retags = "<li class='retags-wrapper'>" + $retags[0].outerHTML + "</li>";
```

Comment:
I agree, and tried for a good long time to make it work with pure CSS and didn't have any luck; if someone else wants to take a crack at it they're more than welcome and maybe we can pull this flag back out.